packages/chains/src/chains/chainMetadata/index.ts

```diff
@@ -160,3 +236,31 @@ export const chains: {
     nativeToken: eth,
   },
 };
+
+const MS_PER_DAY = 1000 * 60 * 60 * 24;
+
+export const getBlockTimeByChainId = (chainId: ChainId, targetTimestamp = new Date().getTime()) => {
+  const targetChain = chains?.[chainId ?? ''];
```

Comment:
`chains` and `chainId` can not be `undefined`, so the optional chaining and the fallback are unnecessary.

```suggestion
  const targetChain = chains[chainId];
```